### DISCUSSION TITLE
How to get English words instead of Arabic numerals while transcribing English speech?

### DISCUSSION THREAD

**Discussion by NanaOrangejuice** [openai/whisper#935] [Q&A] [answered]
I used transcribe.py to infer English speech, and got results containing Arabic numerals, such as "in 7 days". Can I get "in seven days" in transcribing?

**Comment by whicks1** [comment]
Try the initial prompt flag to nudge it or post process the output with something like the num2words.

**Reply by weihanchen** [reply]
how to do with prompt?

**Reply by glangford** [reply, marked as answer]
There is also this approach, see the following discussion:
* https://github.com/openai/whisper/discussions/1041

For `initial_prompt`, see for example
* https://github.com/openai/whisper/discussions/355

```
initial_prompt: Optional[str]
        Optional text to provide as a prompt for the first window. This can be used to provide, or
        "prompt-engineer" a context for transcription, e.g. custom vocabularies or proper nouns
        to make it more likely to predict those word correctly.
```
